.circleci/config.yml

```diff
@@ -64,379 +67,255 @@ jobs:
-              -f docker/compose/ci/postgresql/docker-compose.yml \
-              --project-directory . \
-              run --rm --no-deps app \
-                flake8
-
-  lint-back-pylint:
-    machine: true
-    working_directory: ~/fun
-    steps:
-      - checkout
-      - restore_cache:
-          keys:
-            - docker-debian-images-ci-{{ .Revision }}
+          command: ~/.local/bin/isort --recursive --check-only .
       - run:
-          name: Load images to docker engine
-          command: |
-            docker load < docker/images/dev/richie.tar
+          name: Lint code with black
+          command: ~/.local/bin/black src/richie/apps src/richie/plugins sandbox tests --check
       - run:
           name: Lint code with pylint
-          command: |
-            docker-compose \
-              -p richie-ci \
-              -f docker/compose/ci/postgresql/docker-compose.yml \
-              --project-directory . \
-              run --rm --no-deps app \
-                pylint src/richie/apps src/richie/plugins sandbox tests
-
-  lint-back-black:
-    machine: true
-    working_directory: ~/fun
-    steps:
-      - checkout
-      - restore_cache:
-          keys:
-            - docker-debian-images-ci-{{ .Revision }}
-      - run:
-          name: Load images to docker engine
-          command: |
-            docker load < docker/images/dev/richie.tar
-      - run:
-          name: Lint code with black
-          command: |
-            docker-compose \
-              -p richie-ci \
-              -f docker/compose/ci/postgresql/docker-compose.yml \
-              --project-directory . \
-              run --rm --no-deps app \
-                black src/richie/apps src/richie/plugins sandbox tests --check
+          command: ~/.local/bin/pylint src/richie/apps src/richie/plugins sandbox tests
 
   # Restore front & back POT files containing strings to translate and upload them to our
   # translation management tool
   upload-i18n-strings:
-    machine:
-      docker_layer_caching: true
+    docker:
+      - image: fundocker/crowdin:2.0.27
     working_directory: ~/fun
     steps:
       - checkout
       - attach_workspace:
           at: ~/fun
       - run:
           name: upload files to crowdin
-          command: |
-            docker-compose \
-              -p richie-test \
-              -f docker/compose/ci/postgresql/docker-compose.yml \
-              --project-directory . \
-              run --rm --no-deps crowdin \
-                -c crowdin/config.yml upload sources
+          command: crowdin -c crowdin/config.yml upload sources
 
   test-back-mysql:
-    machine: true
+    docker:
+      - image: circleci/python:3.7-stretch
+        environment:
+          DJANGO_SETTINGS_MODULE: settings
+          DJANGO_CONFIGURATION: Test
+          DJANGO_SECRET_KEY: ThisIsAnExampleKeyForTestPurposeOnly
+          PYTHONPATH: /home/circleci/fun/sandbox
+          RICHIE_ES_HOST: localhost
```

Comment:
then maybe we should put localhost as the default ES host WDYT?

Reply:
🤔 not sure we often map local ports to an ES cluster.